package.json

```diff
@@ -162,7 +162,7 @@
     "babel-runtime": "^6.26.0",
     "bunyan": "^1.8.12",
     "chai": "^4.1.2",
-    "chromedriver": "^2.33.2",
+    "chromedriver": "2.33.2",
```

Comment:
why?

Reply:
Similar to [the situation with `sequelize`](https://github.com/plotly/falcon-sql-client/pull/488#issuecomment-407094951), Chromedriver pulled in a minor patch which was throwing on CI (for example [here](https://circleci.com/gh/plotly/falcon-sql-client/5426)). Freezing the version at `2.33.2` resolved the error.